MSAL/src/native_auth/network/responses/validator/token/validated_response/MSALNativeAuthTokenValidatedResponse.swift

```diff
@@ -26,6 +26,7 @@
 
 enum MSALNativeAuthTokenValidatedResponse {
     case success(MSIDTokenResponse)
+    case strongAuthRequired(continuationToken: String)
```

Comment:
Strong auth required is not exactly an error (it depends on tenant/app configuration), but technically it is: 400 bad request -> `invalid_grant/mfa_required` (although it has a continuation token). Should we consider it as a new kind of response (are we going to have more different types in the future, other than success/error?), or deal with it as an error? I tend to prefer the proposed approach, but would like to hear from you @nilo-ms.

Reply:
Fair question Marcos. I don't have a strong opinion here, but I mainly created a new type of result because the continuation token in this type of result should not be nil. While, if we consider this as error the continuation token can be nil because is optional. This is the only reason why I created a dedicated response